### ISSUE TITLE
Source/Destination training data

### ISSUE THREAD

**Issue by mdeeds** [mdeeds/genie#3]
Use the Source/Destination data saved in the cache for training the model.

This can probably be done asynchronously and continuously.  We should probably stop training the model as soon as it gets 100% on all training data.  Don't continue to push labels all the way down to 1 or 0.  Instead, terminate when all classifications on the training set are correct and outside the 0.3 to 0.7 range.

**Comment by mdeeds** [comment]
Fixed in 931f8563707d12506ea55668f2db91b7ca1b933b
